src/Components/Header/Header.module.css

```diff
@@ -1,7 +1,58 @@
 .header {
   display: flex;
   align-items: center;
-  width: 100%;
-  padding: 2rem;
+  padding: 0 8rem;
+  height: 5.2rem;
   gap: 2rem;
+  margin: 0 0 2.375rem;
+  border-bottom: solid 3px #f6f6f6;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #ffff;
```

Comment:
- [ ] Great job 👍 . Kindly make sure you give your design the correct `margin` and `padding` as seen in the provided design 👏 

Provided Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/201355606-820f9111-8668-4e9c-bd0e-395d21e545a5.png)  |  ![image](https://user-images.githubusercontent.com/74814780/201355367-64a30b30-15fa-4bff-8565-fe3d00a6d836.png)